src/components/v2/TextField/TextField.tsx

```diff
@@ -16,7 +15,7 @@ export interface ITextFieldProps extends Omit<InputHTMLAttributes<HTMLInputEleme
 }
 
 export const TextField: React.FC<ITextFieldProps> = ({
-  css,
+  className,
```

Comment:
So I thought the same as you, but it turns out even when you pass a `css` prop to the component, emotion will just pass down a class instead, which is why we need to pass down `className` in order to make the component's style customizable through the `css` prop.

Reply:
Per the [documentation](https://emotion.sh/docs/composition) we can do this `css={[styles.button, css]}`
This isn't working? 

Emotion should take these styles add a `className` and  render a style tag on the dom. The `className` that emotion generates should be unique which is nice because it can help avoid styles being unintentionally applied to other elements

Reply:
Unfortunately it doesn't :/
The `css` property comes out as `undefined` on the button when it's being passed down.